src/SFA.DAS.Data.Application/Commands/CreateCommitmentStatistics/CreateCommitmentStatisticsCommandResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SFA.DAS.Data.Application.Interfaces;
+
+namespace SFA.DAS.Data.Application.Commands.CommitmentRdsStatistics
```

Comment:
"Rds" is still in the namespace "CommitmentRdsStatistics" - remove this?

Reply:
Done